app/styles/main.scss

```diff
@@ -50,27 +54,37 @@
   }
 }
 
+.footer {
+  background-color: $color-footer;
+  & > .bar-right {
+    margin: 30px;
+  }
+}
+
 .visible-lg {
   visibility: hidden;
 }
 
 @media only screen and (min-width: 768px) {
-  .content > div {
-    display: flex;
-    flex-direction: row;
+  .content {
+    & > div {
+      display: flex;
+      flex-direction: row;
+    }
+    .width-main {
+      width: 75%;
+    }
+    .width-sidebar {
+      width: 25%;
+    }
+    .width-full {
```

Comment:
Why is this class needed. `div` being a block element would be 100% in width unless specified otherwise.